Add more information about slow bluetooth connections (#27869)
home-assistant/home-assistant.github.io: source/_integrations/bluetooth.markdown

### Known working adapters

<div class='note'>

> Known working adapters list adapters that do not meet high-performance requirements but will generally work. These adapters vary widely in performance and may take as long as thirty seconds or more to establish a connection.

Author: ```suggestion
Known working adapters list adapters that do not meet high-performance requirements but will generally work. These adapters vary widely in performance and may take as long as thirty seconds or more to establish a connection. These adapters may also miss advertisements such as button presses or temperature updates.
```

> Known working adapters list adapter that do not meet high performance requirements, but will generally work. These adapters vary widely in performance, and may take as long as thirty seconds or more to establish a connection.

Author: ```suggestion
Known working adapters list adapters that do not meet high-performance requirements but will generally work. These adapters vary widely in performance and may take as long as thirty seconds or more to establish a connection.
```

## Troubleshooting

### Improving connection times

> The connection time and performance varies greatly based on the bluetooth adapter. Below adapters are listed from best performing to worst performing:

Author: ```suggestion
The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:
```

> The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best performing to worst performing:

Author: ```suggestion
The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best-performing to worst-performing:
```

> The connection time and performance vary greatly based on the Bluetooth adapter. The below adapters are listed from best-performing to worst-performing:

Author: ```suggestion
The connection time and performance vary greatly based on the Bluetooth adapter and interference. The below adapters are listed from best-performing to worst-performing:
```

- Bluetooth proxies running ESPHome 2023.6.0 or later with passive scanning

> - USB High performance adapter with passive scanning

Author: ```suggestion
- [USB High performance adapter](#known-working-high-performance-adapters) with passive scanning
```

> - [USB High performance adapter](#known-working-high-performance-adapters) with passive scanning

Author: ```suggestion
- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
```

> - [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)

Author: ```suggestion
- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

- Bluetooth proxies running ESPHome 2023.6.0 or later with active scanning

> - USB High performance adapter with active scanning

Author: ```suggestion
- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
```

> - Bluetooth proxies running ESPHome 2023.6.0 or later with passive scanning
> - [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
> - Bluetooth proxies running ESPHome 2023.6.0 or later with active scanning

Author: ```suggestion
- [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
- [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

> - Onboard high performance adapter with passive scanning

Author: ```suggestion
- Onboard high performance adapter with [passive scanning](#passive-scanning)
```

> - Onboard high performance adapter with [passive scanning](#passive-scanning)
> - Onboard high performance adapter with active scanning

Author: ```suggestion
- [Onboard high performance adapter](#cypress-based-adapters) with [passive scanning](#passive-scanning)
- [Onboard high performance adapter](#cypress-based-adapters) with active scanning
```

> - [Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)

Author: ```suggestion
- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

> - Known working adapters with passive scanning

Author: ```suggestion
- Known working adapters with [passive scanning](#passive-scanning)
```

> - Known working adapters with [passive scanning](#passive-scanning)
> - Known working adapters with active scanning

Author: ```suggestion
- [Known working adapters](#known-working-adapters) with [passive scanning](#passive-scanning)
- [Known working adapters[(#known-working-adapters) with active scanning
```

> - [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
> - [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
> - [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
> - [USB High performance adapter](#known-working-high-performance-adapters) with active scanning

Author: ```suggestion
- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)
- [WiFi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning
- [WiFi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

> - [Known working adapters[(#known-working-adapters) with active scanning

Author: ```suggestion
- [Known working adapters](#known-working-adapters) with active scanning
```

## Troubleshooting

> ### Improving connection times

Reviewer: The title "Improving connection times" and the PR message suggest that there are instructions on how to improve connection times. 
So the measure to take is to use a different device, one that is higher up in the list?

The connection time and performance vary greatly based on the Bluetooth adapter and interference. The below adapters are listed from best-performing to worst-performing:

- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [USB High performance adapter](#known-working-high-performance-adapters) with [passive scanning](#passive-scanning)

> - [WiFi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)

Reviewer: ```suggestion
- [Wi-Fi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [passive scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```

- [Ethernet-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
- [USB High performance adapter](#known-working-high-performance-adapters) with active scanning

> - [WiFi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)

Reviewer: ```suggestion
- [Wi-Fi-connected Bluetooth proxies](#remote-adapters-bluetooth-proxies) running ESPHome 2023.6.0 or later with [active scanning](https://esphome.io/components/esp32_ble_tracker.html#configuration-variables)
```
